Classes/Issues/PullRequest+IssueType.swift

```diff
@@ -12,6 +12,10 @@ import StyledTextKit
 
 extension IssueOrPullRequestQuery.Data.Repository.IssueOrPullRequest.AsPullRequest: IssueType {
 
+    var viewerSubscribed: Bool {
+        return viewerSubscription?.rawValue != "IGNORED"
```

Comment:
nit: We should use a constant for this value here: `SubscriptionState.ignored.rawValue`